### DISCUSSION TITLE
Parsing multiple feat: or fix: from the commit message

### DISCUSSION THREAD

**Discussion by suhas-cashfree** [orhun/git-cliff#665] [Q&A]
### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

When we pass multiple feat: or fix: in one single commit, it doesn't parse. Is it possible to add this functionality

### Desired solution

Parsing multiple feat:

### Alternatives considered

Splitting feat: to multiple commits

**Comment by orhun** [comment]
Hello, this does not sound like a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/), can you share an example?

For more information:

- https://git-cliff.org/docs/configuration/git#conventional_commits
- https://git-cliff.org/docs/templating/context#conventional-commits
